core/src/main/java/com/google/cloud/sql/core/CloudSqlInstance.java

```diff
@@ -448,9 +450,9 @@ private static Certificate createCertificate(String cert) throws CertificateExce
    * Checks for common errors that can occur when interacting with the Cloud SQL Admin API, and adds
    * additional context to help the user troubleshoot them.
    *
-   * @param ex exception thrown by the Admin API request
+   * @param ex           exception thrown by the Admin API request
```

Comment:
Was this intentional?

Reply:
No, my code formatter automatically does this (I have it set to Google Style).